io-engine/tests/ftl_mount_fs.rs

```diff
@@ -0,0 +1,141 @@
+//TODO: with spdk v24.09 ftl devices support variable sector size emulation. At that point we can
+//enable testing with aio devices, which do not require a metadata section in the LBA format.
+//For now we are hiding this test case behind the `ignore` attribute.
```

Comment:
```suggestion
//! TODO: with spdk v24.09 ftl devices support variable sector size emulation. At that point we can
//! enable testing with aio devices, which do not require a metadata section in the LBA format.
//! For now we are hiding this test case behind the `ignore` attribute.
```

Reply:
Could you please point me to the commit which enables this emulation, just for my curiosity? :)

Reply:
The commit which enables this:
https://review.spdk.io/gerrit/c/spdk/spdk/+/19628/82

You can also look at the `Submitted together` commits that did the actual work :)